topic04-git-introduction/book-git-tutorial/03.Rebasing-Without-Conflicts.md

```diff
@@ -18,5 +19,15 @@ git rebase --abort
 **git rebase --abort** : kills the process completely, allowing you to start again.  
 
 ## The process of rebasing
+Before you start rebasing, you want to make sure that your local repo is up to date with the one in the cloud, to do this you type git status in the terminal and see if you're up to date.  
```

Comment:
git status = command, lets use backticks

Reply:
also git status wont tell you if you are out of date with the master of a remote.

Reply:
@dominikkawka ^ The above comments are not implemented